### PR TITLE
Quieten ASP.NET Core merging PRs

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes https://github.com/dotnet/core-eng/issues/11042

Don't @ people with every comment in merge PRs.

Note that this change applies to aspnetcore, extensions and efcore // @ajcvickers

### CODE REVIEW

**Comment by JamesNK** [comment]
@dougbu

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -984,7 +984,8 @@
           "GithubRepoOwner": "aspnet",
           "GithubRepoName": "<trigger-repo>",
           "HeadBranch": "<trigger-branch>",
-          "BaseBranch": "release/3.0"
+          "BaseBranch": "release/3.0",
+          "ExtraSwitches": "-QuietComments"
```
Comment (dougbu) [inline Maestro/subscriptions.json:988]: This is in @vijayrkn's world, not ours'. Sorry if I pointed you here @JamesNK

**Review by dougbu — APPROVED** [review]
WFM but needs @vijayrkn's confirmation as-is

File: Maestro/subscriptions.json (hunk 1 above)
Comment (JamesNK) [reply Maestro/subscriptions.json]: Removed

**Review by dougbu — APPROVED** [review]
Looks good. @dsplaisted @mmitche @wtgodbe could one of you please merge❔

**Pull request merged**